src/models/Factor.js

```diff
@@ -207,7 +207,7 @@ function (Okta, Q, factorUtil, BaseLoginModel) {
         }
 
         // MFA_CHALLENGE
-        else if (this.get('canUseResend') && transaction.resend) {
+        else if (this.get('canUseResend') && !this.get('answer') && transaction.resend) {
```

Comment:
If I use `resend` for validating the answer, backend responses with error:
`{"errorCode":"E0000047","errorSummary":"API call exceeded rate limit due to too many requests.", ...}`
This is why I use resend only to repeat the call or push and not for sending the answer.